src/components/common/ColonyHome/ColonyExtensionsWidget/ColonyExtensionsWidget.tsx

```diff
@@ -1,79 +1,106 @@
-import React from 'react';
-// import { defineMessages, FormattedMessage } from 'react-intl';
+import React, { useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { getExtensionHash } from '@colony/colony-js';
 
-// import Heading from '~core/Heading';
-// import { useColonyExtensionsQuery } from '~data/index';
-// import { MiniSpinnerLoader } from '~core/Preloaders';
-// import extensionData from '~data/staticData/extensionData';
-// import NavLink from '~core/NavLink';
-// import ExtensionStatus from '~dashboard/Extensions/ExtensionStatus';
-// import { useColonyContext } from '~hooks';
+import { MiniSpinnerLoader } from '~shared/Preloaders';
+import Heading from '~shared/Heading';
+import NavLink from '~shared/NavLink';
+import ExtensionStatusBadge from '~common/Extensions/ExtensionStatusBadge';
 
-// import styles from './ColonyExtensions.css';
+import { useColonyContext } from '~hooks';
+import { useGetColonyExtensionsQuery } from '~gql';
+import { InstalledExtensionData } from '~types';
+import { notNull } from '~utils/arrays';
+import { supportedExtensionsConfig } from '~constants/extensions';
 
-const displayName = 'dashboard.ColonyHome.ColonyExtensionsWidget';
+import styles from './ColonyExtensionsWidget.css';
 
-// const MSG = defineMessages({
-//   title: {
-//     id: `${displayName}.title`,
-//     defaultMessage: 'Enabled extensions',
-//   },
-//   loadingData: {
-//     id: `${displayName}.loadingData`,
-//     defaultMessage: 'Loading extensions data ...',
-//   },
-// });
+const displayName = 'common.ColonyHome.ColonyExtensionsWidget';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Enabled extensions',
+  },
+  loadingData: {
+    id: `${displayName}.loadingData`,
+    defaultMessage: 'Loading extensions data ...',
+  },
+});
 
 const ColonyExtensions = () => {
-  // const { colony } = useColonyContext();
+  const { colony } = useColonyContext();
+
+  const { data, loading } = useGetColonyExtensionsQuery({
+    variables: {
+      colonyAddress: colony?.colonyAddress ?? '',
+    },
+    skip: !colony,
+  });
+  const colonyExtensions = data?.getColony?.extensions?.items.filter(notNull);
```

Comment:
Does this take into account @jakubcolony 's latest changes related to refactoring the intalled / installable extensions ?